tools/cjxl_ng_main.cc

```diff
@@ -611,6 +671,18 @@ int main(int argc, char** argv) {
           jxl_encoder_frame_settings, JXL_ENC_FRAME_SETTING_MODULAR_COLOR_SPACE,
           FLAGS_modular_colorspace);
     }
+    if (modular_ma_tree_learning_percent_set) {
+      if (!(-1 <= FLAGS_modular_ma_tree_learning_percent &&
+            FLAGS_modular_ma_tree_learning_percent <= 100)) {
```

Comment:
Values up to 1000 actually have different encoder behavior (it's the percent of samples used in MA tree learning, but also the promille of samples used to determine how to do property quantization when doing sample collection, iirc).